pkg/nsx/services/subnetport/builder.go

```diff
@@ -68,14 +79,14 @@ func (service *SubnetPortService) buildSubnetPort(obj interface{}, nsxSubnetPath
 			TrafficTag:        common.Int64(0),
 			Type_:             String("STATIC"),
 		},
-		Tags:       tags,
+		Tags:       tagsFiltered,
 		Path:       &nsxSubnetPortPath,
 		ParentPath: &nsxSubnetPath,
 	}
 	if appId != "" {
 		nsxSubnetPort.Attachment.AppId = &appId
-		nsxSubnetPort.Attachment.ContextId = &contextID
 	}
+	nsxSubnetPort.Attachment.ContextId = &contextID
 	return nsxSubnetPort, nil
```

Comment:
I saw in your second screenshot of the PR, the port status is "Failed", what's the error message? And is it expected?

Reply:
screenshot updated.